shell/reicast.vcxproj

```diff
@@ -288,8 +288,8 @@
     <ClInclude Include="..\core\hw\sh4\interpr\sh4_opcodes.h" />
     <ClInclude Include="..\core\hw\sh4\modules\ccn.h" />
     <ClInclude Include="..\core\hw\sh4\modules\dmac.h" />
-    <ClInclude Include="..\core\hw\sh4\modules\intc.h" />
```

Comment:
This one was removed in 2007. The file reference somehow still survived (and header files don't get compiled so no compile error)